mongo/options/encryptoptions.go

```diff
@@ -10,20 +10,12 @@ import (
 	"go.mongodb.org/mongo-driver/bson/primitive"
 )
 
-// QueryType describes the type of query the result of Encrypt is used for.
-type QueryType int
```

Comment:
[opt] I suppose you could leave `QueryType` and change its underlying type to `string`.

Also, are users relying on this API/did we release this API already?

Reply:
This was released in beta.

Using `string` is consistent with the `Algorithm` option.